vmslave-config/pipeline/hosts_update/hosts_update.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+import sys
+import jenkins
+import xml.etree.ElementTree as ET
+import argparse
+
+class JenkinsSlaveHostsGenerator():
```

Comment:
Good job. this script will groups jenkins slaves with same ```label``` as a same ```group``` in ansible host file.
so that the ansible can be executed against a ```group```(```label```) for same actions.